whisper/whisperv6/whisper.go

```diff
@@ -696,6 +696,7 @@ func (whisper *Whisper) runMessageLoop(p *Peer, rw p2p.MsgReadWriter) error {
 		}
 		if packet.Size > whisper.MaxMessageSize() {
 			log.Warn("oversized message received", "peer", p.peer.ID())
+			packet.Discard()
```

Comment:
`defer packet.Discard()` ?

Reply:
It will return immediately, I think ```packet.Discard()``` is enough here @gballet